ukrainian/break-through-wall/try-the-taste/continue-eating/wait/wait.md

```diff
@@ -1,7 +1,7 @@
 Ви чекаєте. І тут раптом чуєте гуркіт, обертаєтесь і бачите, що стіна пробита.
 Через отвір видно  постать гнома. Він запрошує вас зайти до себе.
 
-Приймаєте запрошення гнома і [заходите](go-with-dwarf/go-with-dwarf.md) у діру...
-
+Приймаєте запрошення гнома і [заходите](go-with-dwarf/go-with-dwarf.md) у діру.
+Вирішуєте [знову чекати](wait-again/wait-again.md).
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.